### ISSUE TITLE
Madlib packaging issue in Greenplum 7 for madlib 2.1.0 rhel8

### ISSUE THREAD

**Issue by lmugnano4537** [greenplum-db/gpdb#17329]
## Bug Report
I download the madlib2.1.0 from network.pivotal.io (tarball madlib-2.1.0-gp7-rhel8-x86_64.tar.gz).  When I untar this file it creates a subdirectory where it puts the license, readme and what is supposed to be the .gppkg file.  Instead I get these files:

'open_source_license_ Apache MADlib_ 2.1.0_GA.txt'   ReleaseNotes.txt   madlib-2.1.0-gp7-rhel8-x86_64.gppkg.tar.gz

It looks like the file madlib-2.1.0-gp7-rhel8-x86_64.gppkg.tar.gz is really the .gppkg file but the name of the file still has a .tar.gz extension which seems incorrect and is inconsistent of how other versions of madlib on other versions of Greenplum are.

### Greenplum version or build
Greenplum 7.1
Madlib 2.1.0

### OS version and uname -a
CentOS Stream release 8

### Expected behavior
When I untar the downloaded tarball I should get a *.gppkg file that I can install with gppkg

### Actual behavior
The untar is creating a file with extension .gppkg.tar.gz

### Step to reproduce the behavior
tar xzvf madlib-2.1.0-gp7-rhel8-x86_64*.tar.gz -C .
cd to the sub-directory created
look at the files and you'll see there is no .gppkg file

**Comment by Sasasu** [comment]
gppkg in version 7 changed the package format, from now the package is just a simple `tar.gz` file with some metadata inside.
You can install this `.gppkg.tar.gz` file using gppkg in version 7. But I think rename to `.tar.gz.gppkg` may be more meaningful.

@shrakesh

**Comment by lmugnano4537** [comment]
You are correct, gppkg was indeed able to install the oddly named .gppkg file.  I hadn't tried it because I just assumed it had to be a .gppkg file like older releases so I renamed the file before installing it.  I just tried again though with the .gppkg.tar.gz extension and it does work.  I agree it's a bit confusing though
